website/docs/r/policy_share.html.markdown

```diff
@@ -0,0 +1,58 @@
+---
+subcategory: "Multitenancy"
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_share"
+description: A resource to configure a Share.
+---
+
+# nsxt_policy_share
+
+This resource provides a method for the management of a Share.
+This resource is applicable to NSX Policy Manager.
+
+## Example Usage
+
+```hcl
+data "nsxt_policy_project" "test" {
+  display_name = "project1"
+}
+
+resource "nsxt_policy_share" "test" {
+  display_name = "test"
+  description  = "Terraform provisioned Share"
+  shared_with  = [data.nsxt_policy_project.test.path]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `display_name` - (Required) Display name of the resource.
+* `description` - (Optional) Description of the resource.
+* `tag` - (Optional) A list of scope + tag pairs to associate with this resource.
+* `nsx_id` - (Optional) The NSX ID of this resource. If set, this ID will be used to create the resource.
+* `context` - (Optional) The context which the object belongs to
+    * `project_id` - (Required) The ID of the project which the object belongs to
+* `shared_with` - (Required) Path of the context
+* `sharing_strategy` - (Optional) Sharing Strategy. Accepted values: 'NONE_DESCENDANTS', 'ALL_DESCENDANTS'. Default: 'NONE_DESCENDANTS'.
+
+## Attributes Reference
+
+In addition to arguments listed above, the following attributes are exported:
+
+* `id` - ID of the resource.
+* `revision` - Indicates current revision number of the object as seen by NSX-T API server. This attribute can be useful for debugging.
+* `path` - The NSX path of the policy resource.
```

Comment:
If share has a path, would it make sense to use `share_path` rather than `share_id` in `nsxt_policy_shared_resource`? Can `path` be more specific than `id` for the share?

Reply:
That parameter is passed into the SDK API as an id, so in the end even if we pass a path it will be parsed out into an id :)
But I'll change it for consistency as path is used for the shared objects.